torntools/scripts/content/items/ttItems.js

```diff
@@ -757,3 +758,22 @@ function showMissingFlowers() {
 		}
 	}
 }
+
+function showECanGains() {
+	let facECanPerc = parseInt(userdata.faction_perks.filter((x) => /energy drinks/i.test(x)).map((x) => {
+		x.replace(/[^0-9\.]/g, "");
+	})[0]);
+	let jobECanPerc = parseInt(userdata.company_perks.filter((x) => /boost/i.test(x)).map((x) => {
+		x.replace(/[^0-9\.]/g, "");
+	})[0]);
+	doc.findAll("[data-category='Energy Drink']").forEach((ECan) => {
```

Comment:
I'd say this goes against our variable naming convention. 1. this variable starts with a capital letter and 2. this variable represents an element, so it's more readable if the name depicts it.

For example `energyDrinkElement` or `energyDrinkRow`, etc.